tests/drivers/dma/loop_transfer/src/test_dma_loop.c

```diff
@@ -146,11 +146,10 @@ static int test_loop(void)
 	if (transfer_count < TRANSFER_LOOPS) {
 		transfer_count = TRANSFER_LOOPS;
 		TC_PRINT("ERROR: unfinished transfer\n");
-		return TC_FAIL;
 		if (dma_stop(dma, chan_id)) {
 			TC_PRINT("ERROR: transfer stop\n");
-			return TC_FAIL;
 		}
```

Comment:
Is there an interest to keep the "if"?

Reply:
no, except to report the error on dma_stop

Reply:
I don't think there's interest to check behavior after an error